src/components/NavAddToPlaylistIcon.tsx

```diff
@@ -63,6 +63,7 @@ export class NavAddToPlaylistIcon extends React.Component<Props, State> {
           <ActionSheet
             handleCancelPress={this._dismissActionSheet}
             items={actionSheetButtons(episodeId, mediaRefId, navigation, this._dismissActionSheet)}
+            {...(mediaRefId ? { message: 'Do you want to add this episode or clip?' } : {})}
```

Comment:
Could this be `message={mediaRefId ? 'Do you want to add this episode or clip?' : null}` instead? for reading simplicity ofc

Reply:
Done